src/components/sidebar-collection/sidebar-collection.less

```diff
@@ -3,7 +3,9 @@
 @compass-sidebar-default-text-color: @gray5;
 @compass-sidebar-base-background-color: @gray2;
 @compass-sidebar-hover-background-color: @gray1;
+@compass-sidebar-hover-background-color: lighten(hsl(204, 29%, 18%), 5%);
```

Comment:
@imlucas it looks like this works! lmk if you have any other suggestions

Reply:
yay! :shipit: 

![](https://media.giphy.com/media/oJGqcAvg5stm8/giphy.gif)